.changeset/tidy-fans-speak.md

```diff
@@ -0,0 +1,48 @@
+---
+"miniflare": minor
+---
+
+feature: support named entrypoints for `serviceBindings`
+
+This change allows service bindings to bind to a named export of another Worker using designators of the form `{ name: string | typeof kCurrentWorker, entrypoint?: string }`. Previously, you could only bind to the `default` entrypoint. With this change, you can bind to any exported entrypoint.
+
+```ts
+import { kCurrentWorker, Miniflare } from "miniflare";
+
+const mf = new Miniflare({
+	workers: [
+		{
+			name: "a",
+			serviceBindings: {
+				A_RPC_SERVICE: { name: kCurrentWorker, entrypoint: "RpcEntrypoint" },
+				A_NAMED_SERVICE: { name: "a", entrypoint: "namedEntrypoint" },
```

Comment:
Are these service-bindings to entrypoints in the same worker itself?

Reply:
Yup. That is supported 👍